pkg/service/gcp/node.go

```diff
@@ -0,0 +1,166 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"gitlab.com/netbook-devs/spawner-service/pkg/service/common"
+	"gitlab.com/netbook-devs/spawner-service/pkg/service/constants"
+	"gitlab.com/netbook-devs/spawner-service/pkg/service/labels"
+	"gitlab.com/netbook-devs/spawner-service/proto/netbookai/spawner"
+	proto "gitlab.com/netbook-devs/spawner-service/proto/netbookai/spawner"
+	container_proto "google.golang.org/genproto/googleapis/container/v1"
+)
+
+//getNodeFQN retrive fqdn for the node
+func getNodePoolFQN(projectId, location, clusterId, nodeId string) string {
+	return fmt.Sprintf("projects/%s/locations/%s/clusters/%s/nodePools/%s", projectId, location, clusterId, nodeId)
+}
+
+func getDiskType() string {
```

Comment:
can we call this `getDefaultDiskType`?

Reply:
made temporary update, will create open issue for the changes to be made.